lib/core/dialogs/download-zip/download-zip.dialog.stories.ts

```diff
@@ -0,0 +1,86 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { CoreStoryModule } from '../../testing/core.story.module';
+import { MatButtonModule } from '@angular/material/button';
+import {
+    AlfrescoApiService,
+    ContentService,
+    DownloadZipService,
+    NodesApiService
+} from '../../services';
+import { DownloadZipDialogStorybookComponent } from './download-zip.dialog.stories.component';
+import {
+    AlfrescoApiServiceMock,
+    ContentApiMock,
+    DownloadZipMockService,
+    NodesApiMock
+} from '../../mock/download-zip-service.mock';
+import { DownloadZipDialogModule } from './download-zip.dialog.module';
+
+export default {
+    component: DownloadZipDialogStorybookComponent,
+    title: 'Core/Dialog/Download ZIP Dialog',
+    decorators: [
+        moduleMetadata({
+            imports: [
+                CoreStoryModule,
+                DownloadZipDialogModule,
+                MatButtonModule
```

Comment:
can we move it to the DownloadZipDialogModule?

Reply:
In both of stories we use special components built just for Storybook, so MatButtonModule should be here as far as we don't want to put that component into some Module